packages/reporter/src/sessions/sessions.tsx

```diff
@@ -20,9 +20,9 @@ class Sessions extends React.Component<SessionsProps> {
   }
 
   printToConsole = (name) => {
-    const logId = this.props.model[name].id
+    const { id, testId } = this.props.model[name]
```

Comment:
does this need to be type safe? 
```suggestion
    const { id, testId } = this.props.model[name] || {}
```